lib/puppet/type/archive.rb

```diff
@@ -19,60 +19,60 @@
     defaultto(:present)
 
     # The following changes allows us to notify if the resource is being replaced
-    def is_to_s(value)
-      return "(#{resource[:checksum_type]})#{self.provider.archive_checksum}" if self.provider.archive_checksum
+    def s?(value)
```

Comment:
pretty sure we should ignore rubocop since this is overriding is_to_s method specified in ensurable which comes from property.rb:
https://github.com/puppetlabs/puppet/blob/master/lib/puppet/property/ensure.rb#L57
https://github.com/puppetlabs/puppet/blob/master/lib/puppet/property.rb#L349-L351